example-gto/code/.github/workflows/gto-act-on-tags.yml

```diff
@@ -0,0 +1,46 @@
+name: Act on artifact registrations and promotions
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  act:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - run: |
+        set +e
+        # https://stackoverflow.com/questions/10312521/how-to-fetch-all-git-branches
+        git tag | xargs git tag -d
+        for remote in `git branch -r`; do git branch --track ${remote#origin/} $remote; done
+        git fetch --all --prune --tags
+        git pull --all
+        set -e
+    - uses: actions/setup-python@v2
+    - name: set PYSHA
+      run: echo "PYSHA=$(python -VV | sha256sum | cut -d' ' -f1)" >> $GITHUB_ENV
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pre-commit
+        key: pre-commit|${{ env.PYSHA }}|${{ hashFiles('.pre-commit-config.yaml') }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip setuptools wheel
+        pip install -r requirements.txt
```

Comment:
looks like the requirements file is missing here?

Reply:
No, it's there. Earlier is was generated by `pip freeze` and now it's just exists in `code/` folder.